frontend/components/Main/MainOlasBalance.tsx

```diff
@@ -1,16 +1,103 @@
-import { Skeleton } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Flex, Skeleton, Tooltip, Typography } from 'antd';
 import { useMemo } from 'react';
 import styled from 'styled-components';
 
 import { balanceFormat } from '@/common-util/numberFormatters';
 import { UNICODE_SYMBOLS } from '@/constants/unicode';
 import { useBalance } from '@/hooks';
+import { useReward } from '@/hooks/useReward';
 
 import { CardSection } from '../styled/CardSection';
 
+const { Text } = Typography;
 const Balance = styled.span`
   letter-spacing: -2px;
+  margin-right: 4px;
 `;
+const BalanceBreakdown = styled.div`
+  padding: 4px;
+`;
+const BalanceBreakdownLine = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 14px;
+  color: #1f2229;
+
+  > span {
+    background: #fff;
+    z-index: 1;
+    &:first-child {
+      padding-right: 6px;
+    }
+    &:last-child {
+      padding-left: 6px;
+    }
+  }
+
+  &:before {
+    content: '';
+    position: absolute;
+    bottom: 6px;
+    width: 100%;
+    border-bottom: 2px dotted #c2cbd7;
+  }
+
+  &:not(:last-child) {
+    margin-bottom: 8px;
+  }
+`;
+const OVERLAY_STYLE = { maxWidth: '300px', width: '300px' };
+
+const CurrentBalance = () => {
+  const { totalOlasBalance, totalOlasStakedBalance } = useBalance();
+  const { accruedServiceStakingRewards } = useReward();
+
+  const balances = useMemo(() => {
+    return [
+      {
+        title: 'Staked amount',
+        value: balanceFormat(totalOlasStakedBalance ?? 0, 2),
+      },
+      {
+        title: 'Unclaimed rewards',
+        value: balanceFormat(accruedServiceStakingRewards ?? 0, 2),
+      },
+      {
+        title: 'Unused funds',
+        value: balanceFormat(
+          (totalOlasBalance ?? 0) - (totalOlasStakedBalance ?? 0),
+          2,
+        ),
+      },
+    ];
+  }, [accruedServiceStakingRewards, totalOlasBalance, totalOlasStakedBalance]);
+
+  return (
+    <Text type="secondary">
+      Current balance&nbsp;
+      <Tooltip
+        arrow={false}
+        placement="bottom"
+        overlayStyle={OVERLAY_STYLE}
+        title={
+          <BalanceBreakdown>
+            {balances.map((item, index) => (
+              <BalanceBreakdownLine key={index}>
+                <span>{item.title}</span>
+                <span className="font-weight-600">{item.value} OLAS</span>
```

Comment:
strong adds an extra html element to the DOM, plus from the accessibility perspective this might be not correct using "strong", some of screen readers read use strong as an indicator to emphasis pronounced text
trying to avoid that:D

<img width="176" alt="image" src="https://github.com/valory-xyz/olas-operate-app/assets/22725775/b15db631-5591-486b-b1f6-786b527495ed">